### ISSUE TITLE
A11y > Iframe needs a `aria-hidden` attribute

### ISSUE THREAD

**Issue by Blackbaud-TerryHelems** [blackbaud/auth-client#58]
When SPAs run `SkyA11y.run()` in their specs, they generate the following error during e2e tests:
```
error: aXe - [Rule: 'html-has-lang'] <html> element must have a lang attribute - WCAG: wcag311
       Get help at: https://dequeuniversity.com/rules/axe/2.6/html-has-lang?application=webdriverjs
       Elements:
<html>
```
The html rendered in the `SessionWatcher` iframe is the html that does not have the `lang` attribute.  This iframe is hidden and should have the `aria-hidden` attribute set to `true` on it.

**Comment by Blackbaud-TerryHelems** [comment]
Adding the line `iframe.setAttribute('aria-hidden', 'true');` [here](https://github.com/blackbaud/auth-client/blob/2dfa0d0baf3d80e0844229d69ee6e426a61ca924/src/omnibar/omnibar-user-session-watcher.ts#L48) should fix the problem.
